common/views/components/CopyUrl/CopyUrl.js

```diff
@@ -15,7 +15,7 @@ const CopyUrl = ({id, url}: Props) => (
       id='share'
       type='text'
       label='share url'
-      value={url}
+      defaultValue={url}
```

Comment:
This was because value is for dynamic inputs.

Reply:
Good shout – I'd seen that message in the console and had been going to update. Ta.